backend/src/main/java/co/inventorsoft/scripty/service/MockRequestService.java

```diff
@@ -0,0 +1,99 @@
+package co.inventorsoft.scripty.service;
+
+import co.inventorsoft.scripty.exception.ApplicationException;
+import co.inventorsoft.scripty.model.dto.MockRequestDto;
+import co.inventorsoft.scripty.model.entity.MockRequestEntity;
+import co.inventorsoft.scripty.repository.MockRequestRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.UUID;
+
+/**
+ * @author A1lexen
+ */
+
+@Service
+public class MockRequestService {
+    private enum Methods {
```

Comment:
It's better to move this into separate file. It's part of model, but you mix it with logic by putting it inside service